prover/docs/03_launch.md

```diff
@@ -82,8 +70,8 @@ ballpark estimate (useful for local development) is 10 WVGs per prover.
 
```

Comment:
Also add a note here saying that each WVG thread consumes around 10GB of RAM.

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/3134/commits/1187124d2544d5abe0280f7eb5aefca49b18bec9